File: app/src/main/java/com/nimtego/plectrum/presentation/information_view/song/model/SongDetailsModel.java
package com.nimtego.plectrum.presentation.information_view.song.model;

import lombok.Builder;
import lombok.Data;
import lombok.ToString;

@Data
@Builder
@ToString
@Deprecated
public class SongDetailsModel {
    private String songName;
    private String songArtistName;
    private String songArtwork;
    private String songAlbumName;
    private int albumId;
    private String albumArtWorkUrl;
    private Double songPrice;
    private String releaseDate;
    private String wikiInformation;
}

File: app/src/main/java/com/nimtego/plectrum/presentation/old/information_view/song/model/SongDetailsModel.kt
package com.nimtego.plectrum.presentation.old.information_view.song.model

data class SongDetailsModel (
    val songName: String? = null,
    val songArtistName: String? = null,
    val songArtwork: String? = null,
    val songAlbumName: String? = null,
    val albumId: Int = 0,
    val albumArtWorkUrl: String? = null,
    val songPrice: Double? = null,
    val releaseDate: String? = null,
    val wikiInformation: String? = null)